purchase_report_shipping_address/reports/purchase_order_templates.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document_inherit"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//div[@t-if='o.dest_address_id']" position="attributes">
+            <attribute
+                name="t-att-style"
+            >'display: none!important' if o.picking_type_id.warehouse_id.warehouse_address_details else ''</attribute>
+        </xpath>
+        <xpath expr="//div[@t-if='o.dest_address_id']" position="after">
+            <div t-if="o.picking_type_id.warehouse_id.warehouse_address_details">
+                <span
+                    t-field="o.picking_type_id.warehouse_id.warehouse_address_details"
+                />
+            </div>
+        </xpath>
```

Comment:
```suggestion
```
I think this div (i.e. o.dest_address_id is true) is for dropship case and therefore should be intact.  Let me know if I've misunderstood something.